bigbluebutton-html5/imports/ui/components/plugins-engine/data-consumption/manager.tsx

```
@@ -163,14 +180,40 @@
           const HookComponent = hookWithArguments.componentToRender;
           return (
-            <HookComponent
-              numberOfUses={hookWithArguments.numberOfUses}
-              key={makeCustomHookIdentifierFromArgs(hookWithArguments.hookArguments)}
-              hookArguments={hookWithArguments.hookArguments}
-            />
+            <CustomDataConsumptionHooksErrorBoundary
+              hookWithArguments={hookWithArguments}
+              dataConsumptionHook={DataConsumptionHooks.CUSTOM_QUERY}
+              setDataConsumptionHookWithArgumentUtilizationCount={setQueryHookWithArgumentUtilizationCount}
+            >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)